spec/models/item_spec.rb

```diff
@@ -58,10 +58,15 @@
     expect(item).to be_invalid
   end
 
-  xit "won't create an item without a dao" do
+  it "won't create an item without a dao" do
     item.dao = nil
-    # cannot actually be tested since it runs the validation
-    expect(item).to be_invalid
+    item.valid?
+    expect(item.dao).to eq('mugmug')
+  end
+  it "paramaterizing yields dao" do
+    item.dao = nil
+    item.valid?
+    expect(item.to_param).to eq('mugmug')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.